### PR TITLE
Updated test cases for RHOAM v1.15.0

### PR DESCRIPTION
**Pull request by trepel**
# Issue link

https://issues.redhat.com/browse/MGDAPI-3157

# What

Updates to test cases related to RHOAM v1.15.0. The only notable change is in the n10 test case where wording was improved not to mention upgrades at all. That test case is solely about downtime measurement during Quota change. Small issue fixed with the script mentioned there.

# Verification steps

Eye review only.

### CODE REVIEW

File: test-cases/tests/upgrade/n10-verify-quota-feature-upgrade.md [hunk 1]
```diff
@@ -112,3 +114,13 @@ echo rate limit resources: $ratelimit_resources
...
+
+10. Clone [delorean](https://github.com/integr8ly/delorean) repo and run the following command to generate a downtime report using the delorean cli:
+
+```
+cd delorean
+make build/cli
+./delorean pipeline query-report --namespace redhat-rhoam-observability --config-file ./configurations/downtime-report-config-rhoam.yaml -o <output_dir>
+```
+
+There will be a yaml file generated in the output directory.
```
Comment (psturc) [inline test-cases/tests/upgrade/n10-verify-quota-feature-upgrade.md:126]: should there be an instruction about what to do with this yaml file? (upload it somewhere, add it to a jira ticket...)

**Comment by psturc** [comment]
just one comment, otherwise lgtm

File: test-cases/tests/upgrade/n10-verify-quota-feature-upgrade.md (hunk 1 above)
Comment (trepel) [reply test-cases/tests/upgrade/n10-verify-quota-feature-upgrade.md]: As per discussion we agreed that DeLorean report is overkill for Quota check and the workload-web-app would suffice. I removed this part altogether.

**Pull request merged**